kubetest2-tf/deployer/deployer.go

```diff
@@ -201,24 +201,27 @@ func (d *deployer) Up() error {
 		}
 	}
 	inventory := AnsibleInventory{}
+	tfMetaOutput, err := terraform.Output(d.tmpDir, d.TargetProvider)
```

Comment:
tfMetaOutput is of type `map[string]OutputMeta` and needs another round of processing to retrieve the required fields.

```
type OutputMeta struct {
	Sensitive bool            `json:"sensitive"`
	Type      json.RawMessage `json:"type"`
	Value     json.RawMessage `json:"value"`
}
```